new-lamassu-admin/src/components/NotificationCenter/NotificationRow.js

```diff
@@ -73,8 +73,10 @@ const NotificationRow = ({
         </Grid>
       </Grid>
       <Grid item xs={3} style={{ zIndex: 1 }}>
-        {!read && (
-          <div onClick={() => onClear(id)} className={classes.unreadIcon} />
+        {read ? (
+          <div onClick={() => toggleClear(id)} className={classes.readIcon} />
+        ) : (
+          <div onClick={() => toggleClear(id)} className={classes.unreadIcon} />
```

Comment:
change this to how it's done in percentageGraph

Reply:
Is this still relevant? Also might as well just change the className based on the variable:

```js
const clearClasses = {
  [classes.readIcon]: read,
  [classes.unreadIcon]: !read
}
```